### PR TITLE
Update 12-TerryHayesAoc.md

### PR DESCRIPTION
**Pull request by TerryHayesAoc**
hhhh

### CODE REVIEW

File: _slides/12-TerryHayesAoc.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Made caption I am here
```
Comment (selkins13) [inline _slides/12-TerryHayesAoc.md:6]: looks good

File: _slides/12-TerryHayesAoc.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (selkins13) [inline _slides/12-TerryHayesAoc.md:3]: 👍🏻

**Review by selkins13 — APPROVED** [review]
Ship it 🚢

**Comment by TerryHayesAoc** [comment]
hay hay

**Pull request merged**